torch/quantization/quantize.py

```diff
@@ -122,7 +122,7 @@ def add_quant_dequant(module):
         module._modules[name] = add_quant_dequant(child)
     return module
 
-def prepare(model, inplace=False):
+def prepare(model, inplace=False, white_list=None):
```

Comment:
Can you set default argument for white_list=DEFAULT_QCONFIG_PROPAGATE_WHITE_LIST?